worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```diff
@@ -174,7 +172,9 @@
  * using the {@link ChangeSetExtent}.</p>
  */
 @SuppressWarnings({"FieldCanBeLocal"})
+//FAWE start - extends PassthroughExtent > implements Extent
```

Comment:
Add note that editsession should make sure to try to ensure that all edits ultimately go through it else history etc. can have issues (see 2485f5eccc875faff9730b5107e2559612042bad). (PassthroughExtent has some for loops that then delegate to methods editsession overrides)